eclipse-jdt/eclipse.jdt.ui
# Move PropertiesFileEscapes into core

## What it does

This patch moves a small utility class into the org.eclipse.jdt.core.manipulations plugin. This specific file is used by several classes in jdt.ui in the core-refactor source folder. Moving it down will make the migration of the source folder into a core-only plugin go smoother. 

## How to test

Since this is effectively a 'move' operation, as long as the plugin builds and compiles, the patch should likely have no other consequences. 

I have opted not to refactor or change any code within the file for coding standards, so as to make the patch easier to understand and less objectionable. 

## Author checklist
